<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Andromeda AI Object Detection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_islemler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QPushButton" name="bt_images">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Objects From Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_camera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Objects From Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_video">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Objects From Video</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
